
{'date': 'Mon Sep 07 2020 16:34:20.353 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 7
  EQUIPAMENTO="'
                ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Mon Sep 07 2020 16:38:51.253 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 11
  alunos= Elemento(GRUPO.ESTUDANTES)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Mon Sep 07 2020 16:39:54.780 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.vitollino.main import Cena, Elemento, Style, Sala
ImportError: cannot import name 'Style'

ImportError
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.vitollino.main import Cena, Elemento, Style, Sala
'''},
{'date': 'Mon Sep 07 2020 16:40:29.11 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    from _spy.vitollino.main import inventario as inv
ImportError: cannot import name 'inventario'

ImportError
Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    from _spy.vitollino.main import inventario as inv
'''},
{'date': 'Mon Sep 07 2020 16:42:23.461 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 5
    GRUPO.ESTUDANTES="https://static.lenovo.com/br/landings/estudantes/assets/img/lenovo-educational-banner.png"
NameError: name 'GRUPO' is not defined
'''},
{'date': 'Mon Sep 07 2020 16:43:39.547 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 5
    GRUPO.ESTUDANTES="https://static.lenovo.com/br/landings/estudantes/assets/img/lenovo-educational-banner.png"
NameError: name 'GRUPO' is not defined
'''},
{'date': 'Mon Sep 07 2020 16:44:08.965 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 10
    class celulas:
  module <module> line 19
    fala.vai()
NameError: name 'fala' is not defined
'''},
{'date': 'Mon Sep 07 2020 16:44:14.528 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 10
    class celulas:
  module <module> line 19
    fala.vai()
NameError: name 'fala' is not defined
'''},
{'date': 'Mon Sep 07 2020 16:45:22.956 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 10
    class celulas:
  module <module> line 19
    fala.vai()
NameError: name 'fala' is not defined
'''},
{'date': 'Mon Sep 07 2020 16:46:33.565 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 11
    class celulas:
  module <module> line 20
    fala.vai()
NameError: name 'fala' is not defined
'''},
{'date': 'Mon Sep 07 2020 17:13:36.9 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 11
    class celulas:
  module <module> line 20
    fala.vai()
NameError: name 'fala' is not defined
'''},
{'date': 'Tue Sep 08 2020 03:52:33.68 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 11
    class celulas:
  module <module> line 20
    fala.vai()
NameError: name 'fala' is not defined
'''},
{'date': 'Tue Sep 08 2020 03:52:41.483 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 11
    class celulas:
  module <module> line 20
    fala.vai()
NameError: name 'fala' is not defined
'''},
{'date': 'Mon Sep 14 2020 15:24:53.460 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': ''''''},
{'date': 'Mon Sep 14 2020 15:25:06.148 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': ''''''},
{'date': 'Mon Sep 14 2020 15:25:28.751 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': ''''''},
{'date': 'Mon Sep 14 2020 15:25:51.251 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': ''''''},
{'date': 'Mon Sep 14 2020 15:27:23.97 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': ''''''},
{'date': 'Mon Sep 14 2020 21:19:37.659 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  maria.entra(fiocruz, Texto"acho que eu deveria jogar o pacote fora,mas se eu olhar antes e depois jogar?")
                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:09:19.365 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  maria.entra(fiocruz, Texto"acho que eu deveria jogar o pacote fora,mas se eu olhar antes e depois jogar?")
                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:10:36.134 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  maria.entra(fiocruz, Texto"acho que eu deveria jogar o pacote fora,mas se eu olhar antes e depois jogar?")
                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:11:16.273 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  maria.entra(fiocruz, Texto"acho que eu deveria jogar o pacote fora,mas se eu olhar antes e depois jogar?")
                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:11:56.262 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  maria.entra(fiocruz, Texto"acho que eu deveria jogar o pacote fora,mas se eu olhar antes e depois jogar?")
                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:12:21.602 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 45
  emaria.entra(fiocruz)=efala
                       ^
SyntaxError: can't assign to function call 
'''},
{'date': 'Tue Sep 15 2020 03:12:31.393 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 63
  laboratorio= Cena(img =  LABORATORIO)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 15 2020 03:12:53.974 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 61
  """
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:13:14.489 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 62
  """class Fase1():
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:14:02.686 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 62
  """class Fase1():
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:14:13.470 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 64
  """class Fase1():
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:14:51.835 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 63
  """
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:15:52.276 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 63
  """
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:17:10.891 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 63
  """
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 15 2020 03:18:21.711 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 80
  """
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Sep 15 2020 03:18:38.197 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: innermost.C.tree[0].args.indexOf is not a function>
'''},
{'date': 'Tue Sep 15 2020 16:02:03.98 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: innermost.C.tree[0].args.indexOf is not a function>
'''},
{'date': 'Tue Sep 15 2020 16:07:38.75 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: innermost.C.tree[0].args.indexOf is not a function>
'''},
{'date': 'Tue Sep 15 2020 19:08:53.18 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: innermost.C.tree[0].args.indexOf is not a function>
'''},
{'date': 'Tue Sep 15 2020 19:10:12.629 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: innermost.C.tree[0].args.indexOf is not a function>
'''},
{'date': 'Tue Sep 15 2020 19:30:29.402 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: innermost.C.tree[0].args.indexOf is not a function>
'''},
{'date': 'Wed Sep 16 2020 00:15:48.304 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    class apresentacao():
  module <module> line 65
    cartaz= Elemento(img= CARTAZ, Texto = "para se usar o microscópio faça isso...")
NameError: name 'CARTAZ' is not defined
'''},
{'date': 'Wed Sep 16 2020 00:17:52.919 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 67
  cartaz.vai=Texto(cartaz, "para se usar o microscópio faça isso...")
  ^
IndentationError: expected an indented block
'''},
{'date': 'Wed Sep 16 2020 00:18:26.540 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 73
    apresentacao()
  module <module> line 25
    maria= Elemento(img= MARIA, texto= " olá me chamo Maria e amo estudar biologia e lutar pelos direitos das mulheres",cena=a_cena, x=460, y=250, w=150, h=80)
NameError: name 'a_cena' is not defined
'''},
{'date': 'Wed Sep 16 2020 00:21:20.135 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 62
  maria.vai= Texto(casa,= " olá me chamo Maria e amo estudar biologia e lutar pelos direitos das mulheres")
                                                                                                           ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Sep 16 2020 00:21:42.843 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 76
    apresentacao()
  module <module> line 25
    maria= Elemento(img= MARIA,cena=a_cena, x=460, y=250, w=150, h=80)
NameError: name 'a_cena' is not defined
'''},
{'date': 'Wed Sep 16 2020 00:22:10.684 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 76
    apresentacao()
  module <module> line 33
    maria.entra(rua, texto= "estou indo para FIOCRUZ, acho que vou conseguir um estágio lá, uip")
TypeError: entra() got an unexpected keyword argument 'texto'
'''},
{'date': 'Wed Sep 16 2020 00:24:32.414 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 34
  rua2=Cena(img= RUA2)
      ^
SyntaxError: can't assign to function call 
'''},
{'date': 'Wed Sep 16 2020 00:25:55.44 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 80
    apresentacao()
  module <module> line 37
    maria.entra(rua2, texto= "parece uma pessoa estranha, não quero aceitar nada dela")
TypeError: entra() got an unexpected keyword argument 'texto'
'''},
{'date': 'Wed Sep 16 2020 00:36:44.877 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 91
    apresentacao()
  module <module> line 41
    rua2.direta=fiocruz
UnboundLocalError: local variable 'fiocruz' referenced before assignment
'''},
{'date': 'Wed Sep 16 2020 00:38:14.668 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 91
    apresentacao()
  module <module> line 55
    microscopio = Elemento(img = microscopio)
UnboundLocalError: local variable 'microscopio' referenced before assignment
'''},
{'date': 'Wed Sep 16 2020 00:39:01.175 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 91
    apresentacao()
  module <module> line 55
    microscopio = Elemento(img = MICROSCOPIO)
NameError: name 'MICROSCOPIO' is not defined
'''},
{'date': 'Wed Sep 16 2020 00:39:22.434 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 92
    apresentacao()
  module <module> line 46
    microscopio = Elemento(img = MICROSCOPIO)
NameError: name 'MICROSCOPIO' is not defined
'''},
{'date': 'Wed Sep 16 2020 00:40:13.396 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 92
    apresentacao()
  module <module> line 47
    cartaz= Elemento(img= CARTAZ)
NameError: name 'CARTAZ' is not defined
'''},
{'date': 'Wed Sep 16 2020 20:19:04.908 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 111
  def entrou_procarionte(*_):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Sep 16 2020 20:19:26.340 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 111
  def entrou_procarionte(*_):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Sep 16 2020 20:20:00.237 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 135
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Wed Sep 16 2020 20:21:39.234 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 134
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Wed Sep 16 2020 20:21:56.946 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 135
    Fase1()
  module <module> line 101
    procarionte= Elemento(img= PROCARIONTE)
NameError: name 'PROCARIONTE' is not defined
'''},
{'date': 'Wed Sep 16 2020 20:54:05.872 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 134
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Thu Sep 17 2020 20:34:44.351 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 134
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Fri Sep 18 2020 18:59:52.737 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 134
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Fri Sep 18 2020 18:59:56.144 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 134
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Fri Sep 18 2020 18:59:59.334 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 134
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Fri Sep 18 2020 19:00:02.374 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 134
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Fri Sep 18 2020 19:00:05.878 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 134
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Fri Sep 18 2020 19:00:09.757 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 134
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Fri Sep 18 2020 19:00:12.717 GMt-0300 (Hora padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 134
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Mon Sep 21 2020 21:30:38.924 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 96
    apresentacao()
  module <module> line 60
    celula= Cena(img= CELULA)#QUANDO CLICA NO MICRÓSCÓPIO IRÁ PARA ESSA CENA
NameError: name 'CELULA' is not defined
'''},
{'date': 'Mon Sep 21 2020 21:32:38.302 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 96
    apresentacao()
  module <module> line 60
    celula= Cena(img= CELULA)#QUANDO CLICA NO MICRÓSCÓPIO IRÁ PARA ESSA CENA
NameError: name 'CELULA' is not defined
'''},
{'date': 'Mon Sep 21 2020 22:01:33.280 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 143
  self.npc.vai=Texto(self.celula= "  vc foi trazida para outra dimensão, a dimensão microscópica das células" 
                                ^
SyntaxError: keyword can't be an expression
'''},
{'date': 'Mon Sep 21 2020 22:02:34.72 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 143
  self.npc.vai=Texto(self.celula= "  vc foi trazida para outra dimensão, a dimensão microscópica das células.Para sair, vc terá  que vencer alguns enigmas e desafios, relacionados ao mundo das células ou ficará presa aqui para sempre.").vai 
                                ^
SyntaxError: keyword can't be an expression
'''},
{'date': 'Mon Sep 21 2020 22:04:30.340 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 143
  self.npc.vai=Texto(self.celula= "  vc foi trazida para outra dimensão, a dimensão microscópica das células.Para sair, vc terá  que vencer alguns enigmas e desafios, relacionados ao mundo das células ou ficará presa aqui para sempre.").vai 
                                ^
SyntaxError: keyword can't be an expression
'''},
{'date': 'Mon Sep 21 2020 22:13:36.397 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 143
  self.npc.vai=Texto(self.celula= "  vc foi trazida para outra dimensão, a dimensão microscópica das células.Para sair, vc terá  que vencer alguns enigmas e desafios, relacionados ao mundo das células ou ficará presa aqui para sempre.").vai 
                                ^
SyntaxError: keyword can't be an expression
'''},
{'date': 'Mon Sep 21 2020 22:27:16.151 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 143
  self.npc.vai=Texto(self.celula= """  vc foi trazida para outra dimensão, a dimensão microscópica das células.
                                ^
SyntaxError: keyword can't be an expression
'''},
{'date': 'Tue Sep 22 2020 09:47:49.985 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 96
    apresentacao()
  module <module> line 60
    celula= Cena(img= CELULA)#QUANDO CLICA NO MICRÓSCÓPIO IRÁ PARA ESSA CENA
NameError: name 'CELULA' is not defined
'''},
{'date': 'Tue Sep 22 2020 09:48:35.144 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 148
    laboratorio.vai()
NameError: name 'laboratorio' is not defined
'''},
{'date': 'Tue Sep 22 2020 09:56:13.247 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 149
    Fase1().vai()
  module <module> line 103
    self.procarionte= Elemento(img= PROCARIONTE)
NameError: name 'PROCARIONTE' is not defined
'''},
{'date': 'Tue Sep 22 2020 09:59:22.24 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 150
    Fase1().vai()
  module <module> line 112
    self.npc= Elemento(img= NPC,Texto = "você precisa descobrir quem é procarionte para a próxima fase")#resposta certa é a bactéria
NameError: name 'NPC' is not defined
'''},
{'date': 'Tue Sep 22 2020 11:08:26.494 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 153
    Fase1().vai()
  module <module> line 113
    self.npc_texto= Texto(self.laboratorio,"você precisa descobrir quem é procarionte para a próxima fase", foi= self.entra_procarionte)#resposta certa é a bactéria
AttributeError: 'Fase1' object has no attribute 'entra_procarionte'
'''},
{'date': 'Tue Sep 22 2020 15:06:06.840 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''<class 'apresentacao'>

RuntimeError: too much recursion
  module '$exec_320' line 104
        self.procarionte= Elemento(img= PROCARIONTE)
'''},
{'date': 'Tue Sep 22 2020 17:28:51.744 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 97
    apresentacao().vai()
AttributeError: 'apresentacao' object has no attribute 'vai'
'''},
{'date': 'Tue Sep 22 2020 18:16:42.533 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 117
    apresentacao().vai()
  module <module> line 70
    self.microscopio.texto.foi=self.entrou_maria
AttributeError: 'apresentacao' object has no attribute 'microscopio'
'''},
{'date': 'Tue Sep 22 2020 18:18:56.30 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 113
  """arrastar e colocar a lâmina no microscópio e então aparece um um ciclone que a leva para dentro da célula, 
  ^
IndentationError: unexpected indent
'''},
{'date': 'Tue Sep 22 2020 18:19:53.293 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 115
    apresentacao().vai()
  module <module> line 70
    self.casa.texto.foi=self.entrou_maria
AttributeError: 'Cena' object has no attribute 'texto'
'''},
{'date': 'Tue Sep 22 2020 18:21:14.474 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 115
    apresentacao().vai()
AttributeError: 'apresentacao' object has no attribute 'vai'
'''},
{'date': 'Tue Sep 22 2020 18:23:57.845 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 186
    Fase1().vai()
  module <module> line 124
    self.trypanosoma_cruzi= Elemento(img=TRYPANOSOMA_CRUZI, x=200, y=0)
NameError: name 'TRYPANOSOMA_CRUZI' is not defined
'''},
{'date': 'Tue Sep 22 2020 18:25:52.862 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 186
    Fase1().vai()
  module <module> line 124
    self.trypanosoma_cruzi= Elemento(img=TRYPANOSOMA_CRUZI, x=200, y=0)
NameError: name 'TRYPANOSOMA_CRUZI' is not defined
'''},
{'date': 'Tue Sep 22 2020 18:57:41.739 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 39
  self.estranha= Elemento(img= ESTRANHA, x=560,, y=250 texto= "Guarde e proteja esse pacote com sua própria vida")
                                                ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Sep 22 2020 19:39:04.438 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 198
    Fase1().vai()
TypeError: __init__() missing 1 positional argument: maria
'''},
{'date': 'Thu Feb 10 2022 17:27:49.674 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 158
  """
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Feb 10 2022 17:28:04.30 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 199
  def entrou_celula(self, *_):
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed Apr 06 2022 15:38:12.120 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 216
  self.calciforme.vai=Texto(self.laboratorio,
                                                                                                                                                                                                          ^
SyntaxError: invalid syntax : triple string end not found
'''},
{'date': 'Wed Apr 06 2022 15:38:43.940 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 216
  self.calciforme.vai=Texto(self.laboratorio,
                                                                                                                                                                                                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Apr 06 2022 15:55:25.474 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 216
  self.calciforme.vai=Texto(self.laboratorio,
                                                                                                                                                                                                          ^
SyntaxError: invalid syntax : triple string end not found
'''},
{'date': 'Wed Apr 06 2022 15:55:51.635 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 216
  self.calciforme.vai=Texto(self.laboratorio,
                                                                                                                                                                                                          ^
SyntaxError: invalid syntax : triple string end not found
'''},
{'date': 'Wed Apr 06 2022 15:56:06.602 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 216
  self.calciforme.vai=Texto(self.laboratorio,
                                                                                                                                                                                                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Apr 06 2022 15:56:27.903 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 216
  self.calciforme.vai=Texto(self.laboratorio,
                                                                                                                                                                                                          ^
SyntaxError: invalid syntax : triple string end not found
'''},
{'date': 'Wed Apr 06 2022 15:56:36.302 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 216
  self.calciforme.vai= Texto(self.laboratorio,
                                                                                                                                                                                                           ^
SyntaxError: invalid syntax : triple string end not found
'''},